samples/client/petstore/rust/README.md

```diff
@@ -0,0 +1,97 @@
+# Go API client for swagger
```

Comment:
Sure thing, I ought to be able to do it tomorrow. Is the standard to regenerate the petshop code as part of rename PRs?